site/src/docs/components/login/usage.mdx

```diff
@@ -0,0 +1,634 @@
+---
+slug: '/components/login/usage'
+title: 'Login - Usage'
+---
+
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+import ExternalLink from '../../../components/ExternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Table of contents
+
+- <InternalLink href="/components/login/usage#components">Components</InternalLink>
+
+  - <InternalLink href="/components/login/usage#loginprovider">LoginProvider</InternalLink>
+  - <InternalLink href="/components/login/usage#loginbutton">LoginButton</InternalLink>
+  - <InternalLink href="/components/login/usage#LoginCallbackHandler">LoginCallbackHandler</InternalLink>
```

Comment:
Apparently mdx doesn't understand ids if they have capital letters.. This and many after this have to be with small letters. So `href="/components/login/usage#loginbutton"`works as a link but `href="/components/login/usage#LoginCallbackHandler">` doesn't, nothing happens after clicking it.

Reply:
Yeah, all anchors changed to lowercase